\providecommand{\main}{..}
\documentclass[\main/thesis.tex]{subfiles}

\begin{document}

\chapter{Conclusion}
\section{Discussion of Results}
In this thesis we developed a sophisticated cellular automata model for the cancer field effect. The model is an extension to existing cellular automata models \cite{Gerlee} since I include the effect of carcinogens (ethanol and tobacco) on the gene expression of oncogenes and tumour suppressor genes. The gene expression was modelled by a multi-layer neural network, which can be trained once more data is available.

Due to the fact that the chosen genes are not positively mutated towards cancer by ethanol, our model shows that ethanol alone will not cause a field to develop. Nicotine is the most effective carcinogen in our simulations, since it causes the most positive mutations. A field does not develop without carcinogens in the domain, thus verifying the original hypothesis that field cancerization is caused by carcinogenic onslaught, and in particular that it needs consistent carcinogenic onslaught. As well, when investigating cyclic carcinogenic onslaught, we verify that the frequency of carcinogenic onslaught is important, because the less time the carcinogen is in the body, the more time the field takes to develop. 

We determined that grid size does not significantly have an impact on the dynamics of the model, other than that it seems the domain size needs to be a certain size for cancer to thrive. We observed, by tracking the cell lineages, that the model shows that the tumour masses form both from monoclonal and polyclonal origin, with more polyclonal origin than monoclonal.

We demonstrated that when an excision is performed that removes only the tumour cells but leaves the remaining surrounding tissue intact, the cancer recurs faster than when removing the entire field of mutated tissue. When the field is not removed during excision the cancer that recurs is more aggressive than when the field was removed. We found that the more time the field has to develop before excision, the faster and more aggressive the recurrence. 

Our research shows that for heavy smokers, the development of a cancer field is expected, which may or may not lead to cancer. 

% Make sure everything is added here from Keep notes
\section{Future Work}
With regards to the genes, several possible enhancements could be explored. It is possible to increase accuracy of gene expression by accounting for the fact that there are two sets of each gene, positive and negative. A dynamic mutation threshold that would alter on predefined parameters, such as the number of genes that are mutated, or on cell age, could be added. The mutation threshold could also be specific for each gene or each type of gene. In our model we assumed TP53 is related to all other genes and as a result once it is mutated all other genes will become mutated as well. However, it might be better to have an order the gene mutations must occur in such as for RAS to become mutated TP53 must first become mutated. Another possibility would be to only allow a gene to mutate when a related gene is mutated. Currently we consider ten genes in our model, in the future we want to examine using genes specifically associated with cancer formation caused by the carcinogens we consider, especially for ethanol. We could use data from lab experiments to train the gene expression neural network by looking at how each carcinogen effects the expression of each gene. It would be of benefit to include viral infections to the model such as human papillomavirus (HPV) as input to the gene expression neural network. We would want to include genetic precursors towards cancer as input to the gene expression neural network as well. Non-carcinogen mutagen factors to consider adding to the gene expression neural network or a new neural network or through a correlation matrix, include; gender, age and lifestyle. It would be beneficial to be able to use a patient specific data set, such as genetic sequencing to improve accuracy of the values in the gene expression neural network and carcinogen exposure information such as how often the patient smokes.

Future biological mechanisms to add to our model would incorporate wound healing, cell metabolism, micro-environment variables, and cell adhesion. \newline Telomeres are at the end of the DNA strands and with each cell division they get cut shorter, eventually becoming so short that the cell can no longer proliferate and so will enter senescence. Senescent cells are similar to quiescent cells except they canâ€™t perform any actions and at some point undergo apoptosis. Therefore, the model could be enhanced by introducing telomeres. How stem cells are distributed in the domain could be more accurately represented by using a stem cell niche instead of allowing stem cells to distribute randomly in the grid. Currently the model only forms tumour cells from cancer stem cells, whereas we could allow tumour cells to randomly appear, or transition other cell classes into tumour cells. Currently a non-mutated cell class can transition to its associated mutated class as long as it has a certain number of positively mutated genes, it would be more accurate to specify certain genes that must be positively mutated before a transitions to a mutated class is allowed. A Transient amplifying cell (TAC) currently doesn't fully differentiate until it succeeds in producing its allotted number of cell generations, it would be more realistic to have the cell fully differentiate after some rule has been met, such as there being a sufficient number of cells surrounding it or if it has been a certain number of time-steps since first being formed. Carcinogens could be permitted to hinder phenotypic actions from occurring, or allow the direct killing. Carcinogens could also be allowed to weaken cells so that more genetic and phenotypic mutations can be accomplished by other carcinogens. The carcinogens currently spread via carcinogen spatial distributions that don't depend on time, it would be interesting to consider spatial distributions that can vary over time, such as allowing it to increase in size or move location. We currently consider only one type of carcinogen spatial distribution for each carcinogen in a simulation, instead we could allow each carcinogen to have different carcinogen spatial distributions depending on the type of carcinogen. Cyclic carcinogen onslaught could be improved by including a rule for a carcinogen where it is permanently deactivated, for example due to a person quitting smoking. Instead of substituting nicotine as the carcinogen for tobacco we could consider the key carcinogens of tobacco. The chance of occurrence of each phenotypic action could be estimated from lab experiments, by recording how often cells differentiate, proliferate, and move. A case analysis between different types of cancers could be achieved by applying our model to other cancer types. 

Running the model through more simulations and parameters would help to determine a rough estimate of the field size and the parameter values. The ability to tune the grid size and type of lattice to match the tissue and cancer type would be beneficial. More simulations for a broader base of cases could be run such that we are able to determine averages and trends. Some CA parameter analysis could be accomplished by modifying for example the chance a cell moves. Eventually we would want to run the model in a three-dimensional domain. One of the questions we originally wanted to answer was how long it would take for a tumour to become large enough to be detected by physicians, however, we were not able to answer this question due to the size of the cells requiring at least a domain size 1024x1024 to represent the required $1 cm$ detection size. A few simulations at 1024x1024 were run and we found it would take more than 10 years to fill in the space, thus it would take at least 10 years for the tumour to be detectable. Another possibility would be to use a three-dimensional domain, which would require a grid size of only 128x128x64. With regards to the CA we could consider various types of neighbourhoods such as Neumann or extended Moore. All the random samples in the CA are currently taken from a uniform distribution, thus investigation could be conducted on other distributions and potentially a mixture of distributions based upon the use case of the sample point.

With regards to efficiency of running the model, as the complexity increases, the speed of the calculations involved in the gene expression neural network could be improved with linear algebra libraries in CUDA. Using texture memory in the GPU to store cell neighbourhoods would make calculations both faster and easier, as it has faster bandwidth and built in boundaries conditions. The code could be made more cross compatible by allowing parallel computation on the CPU and switching from CUDA to OpenCL. Running the model with cloud computing would be a better option for a program built for hospital use.

\end{document}