\providecommand{\main}{../..}
\documentclass[\main/thesis.tex]{subfiles}

\begin{document}

\section{Cellular Automaton}

As mentioned in the model overview, the six cell types that we consider are 
normal tissue cells (NTC), mutated normal tissue cells (MNTC), normal stem cells 
(NSC), mutated normal stem cells (MNSC), cancer stem cells (CSC), and tumour 
cells (TC). Each cell type is specified in the cellular automata (CA) using a 
numerical value between 0 and 5. More precisely, we have that 0 {=} NTC, 
1 {=} MNTC, 2 {=} NSC, 3 {=} MNSC, 4 {=} CSC, 5 {=} TC.
Since biological cells can move, proliferate, differentiate, and go through 
apoptosis then we must also introduce an empty cell state which is represented 
by the value 6. The cell type in the CA is represented by 
$s(x, y, t) {\in} \{ 0, 1, ..., 6  \}$. Note that when visualizing the CA each 
one value of $s(x, y, t)$ also has a color associated to it.
Each cell in the CA tracks the gene expression of the $G$ genes in a vector 
defined by $\vec{E}(x, y, t) {=} \{ e_j(x, y, t) \}_{j{=}1}^{G}$. The likelihood 
of each phenotype occuring at any given timestep is stored in a vector defined 
by $\vec{P}(x, y, t) {=} \{ p(x, y, t), q(x, y, t), a(x, y, t), d(x, y, t) \}$, 
where $p(x, y, t)$ is the likelihood of proliferation, $q(x, y, t)$ is 
the likelihood of quiescense, $a(x, y, t)$ is the likelihood of 
apoptosis, $d(x, y, t)$ is the likelihood of differentiation. At a timestep in 
the CA a phenotype is randomly chosen to occur based on the likelihoods in 
$\vec{P}(x, y, t)$. As a result of this since we don't want a cell to reproduce 
more than once in a time step then each timestep represents a single cell cycle 
for the type of tissue cell under consideration. The final aspect of the cell 
that is tracked and represented in the overall cell state is the age of the 
cell, $\alpha(x, y, t) {\in} \mathbb{N}$. The state of a cell in the CA is given 
by the vector $S(x, y, t) {=} \{ s(x, y, t), \alpha(x, y, t), \vec{E}(x, y, t), 
\vec{P}(x, y, t) \}$.

\newpage
\subsection{Cell Mutation}

A cell can become mutated through changes in both the likelihood of a phenotype 
occuring and the gene expression. The process of these changes is described in 
this section.

\subsubsection{Gene Mutation}

The chosen $G$ genes are known genes related to the type cancer being studied.
Thus, Each of the $G$ genes is either a tumour supressor gene or an oncogene.
Define the vector $\vec{T} {\in} \{ 0, 1 \}^G$, where each element, 
$\vec{T}_{j}$, represents the type of gene $j$, whereby $T_{j} {=} 0$ represents 
a tumour supressor gene and $T_{j} {=} 1$ represents an oncogene.
As discussed in the previous section the gene $j$ is mutated if
$|e_j(x, y, t)| {\ge} M$. A gene $j$ is positively mutated towards cancer 
(positively mutated) if it is mutated and either it is a tumour supressor gene, 
\ie, $\vec{T}_j {=} 0$, and it's gene expression is downregulated, \ie,
$e(x, y, t)_j < 0$ or it is a oncogene, \ie, $\vec{T}_j {=} 1$, and
it's gene expression is upregulated, \ie, $e(x, y, t)_j > 0$.      
At each time step the gene expression of each gene is updated from the results 
of the gene expression neural network. The changes in the gene expression allow 
the gene to become mutated or even go from mutated to non mutated, \ie, normally 
expressed. 

\subsubsection{Gene Instability}

A gene can influence the expression of another gene in the following way. 
There is a chance, $\tau$, that a positively mutated gene will bring the gene 
expression of a related non positively mutated gene closer to positive 
mutation and a non positively mutated will bring the gene expression of a 
related positively mutated gene further from being positively mutated.
It is important to notice that it doesn't matter if a gene is mutated but only
if it is positively mutated towards cancer. Define the matrix $R {\in} \{ 0, 1 
\}^{G{\times}G}$, where each entry, $R_{ij}$, represents whether gene $i$ is 
related to gene $j$ with $0{=}\text{unrelated}$ and $1{=}\text{related}$. The 
process of a gene $i$ changing the gene expression of another gene $j$ is 
represented by the formula:
\begin{align*}
e(x, y, t)_j {=} \begin{cases}
                   e(x, y, t{-}1)_j {-} v,
                     & z {\le} \tau, R_{ij} {=} 1, \\ 
                     & \vec{T}_i {=} 0,
                       |e(x, y, t{-}1)_i| {\ge} M,
                       e(x, y, t{-}1)_i {<} 0, \\
                     & \vec{T}_j {=} 0,
                       e(x, y, t{-}1)_j {\ge} 0 \\
                   e(x, y, t{-}1)_j {+} v,
                     & z {\le} \tau, R_{ij} {=} 1, \\
                     & \vec{T}_i {=} 0,
                       |e(x, y, t{-}1)_i| {\ge} M,
                       e(x, y, t{-}1)_i {<} 0, \\
                     & \vec{T}_j {=} 1,
                       e(x, y, t{-}1)_j {\le} 0 \\
                   e(x, y, t{-}1)_j {-} v,
                     & z {\le} \tau, R_{ij} {=} 1, \\ 
                     & \vec{T}_i {=} 1,
                       |e(x, y, t{-}1)_i| {\ge} M,
                       e(x, y, t{-}1)_i {>} 0, \\
                     & \vec{T}_j {=} 0,
                       e(x, y, t{-}1)_j {\ge} 0 \\
                   e(x, y, t{-}1)_j {+} v,
                     & z {\le} \tau, R_{ij} {=} 1, \\ 
                     & \vec{T}_i {=} 1,
                       |e(x, y, t{-}1)_i| {\ge} M,
                       e(x, y, t{-}1)_i {>} 0, \\
                     & \vec{T}_j {=} 1,
                       e(x, y, t{-}1)_j {\le} 0 \\
                   e(x, y, t{-}1)_j {+} v,
                     & z {\le} \tau, R_{ij} {=} 1, \\
                     & \vec{T}_i {=} 0,
                       e(x, y, t{-}1)_i {\ge} 0, \\
                     & \vec{T}_j {=} 0,
                       |e(x, y, t{-}1)_j| {\ge} M,
                       e(x, y, t{-}1)_j {<} 0 \\                       
                   e(x, y, t{-}1)_j {-} v,
                     & z {\le} \tau, R_{ij} {=} 1, \\
                     & \vec{T}_i {=} 0,
                       e(x, y, t{-}1)_i {\ge} 0, \\ 
                     & \vec{T}_j {=} 1,
                       |e(x, y, t{-}1)_j| {\ge} M,
                       e(x, y, t{-}1)_j {>} 0 \\
                   e(x, y, t{-}1)_j {+} v,
                     & z {\le} \tau, R_{ij} {=} 1, \\
                     & \vec{T}_i {=} 1,
                       e(x, y, t{-}1)_i {\le} 0, \\ 
                     & \vec{T}_j {=} 0,
                       |e(x, y, t{-}1)_j| {\ge} M,
                       e(x, y, t{-}1)_j {<} 0 \\
                   e(x, y, t{-}1)_j {-} v,
                     & z {\le} \tau, R_{ij} {=} 1, \\
                     & \vec{T}_i {=} 1,
                       e(x, y, t{-}1)_i {\le} 0, \\ 
                     & \vec{T}_j {=} 1,
                       |e(x, y, t{-}1)_j| {\ge} M,
                       e(x, y, t{-}1)_j {>} 0 \\
                 \end{cases}
\end{align*}, where $z {\in} Z(0, 1)$ with $Z$ being a random variable and $v 
{\in} V(c, d)$ with $V$ being a random variable and $(c, d)$ being the range of 
values that the increment can take.  
                      

\subsubsection{Phenotype Mutation}

The likelihood of a phenotype occuring can change at each timestep based upon 
the relationship between what happens to the phenotypes relative to the gene 
expression of a gene. When a gene is mutated there is a chance, $\gamma$, that 
the likelihood of a phenotype occuring, $\vec{P}(x, y, t)_i$, changes. 
Define the matrix $U {\in} \mathbb{R}^{4{\times}G}$, where each entry, 
$U_{ij}$, is a real number that represents an increment to the likelihood of 
phenotype $i$, $\vec{P}(x, y, t)_i$, under the circumstance that gene $j$ is 
mutated and its' expression isupregulated. Similiarly define the matrix
$D {\in} \mathbb{R}^{4{\times}G}$, where each entry, $D_{ij}$, is a real 
number that represents an increment to the likelihood of phenotype $i$,
$\vec{P}(x, y, t)_i$, under the circumstance that gene $j$ is mutated and its' 
expression is downregulated. Through these changes it is insured that the 
likelihood of a phenotype occuring is kept bounded between 0 and 1. Note that 
usually $U{=}{-}D$ since the effects of the upregulation of a gene $j$ has 
the opposite effect on the phenotype $i$ relative to the downregulation of the 
gene. The described update can be written as:
\begin{align*}
\vec{P}(x, y, t)_i {=} \begin{cases}
                         \min(\vec{P}(x, y, t{-}1)_i + U_{ij}, 1),
                           & |e(x, y, t{-}1)_j| {\ge} M,
                             e(x, y, t{-}1)_j {>} 0,
                             z {\le} \gamma,
                             U_{ij} {>} 0 \\
                         \max(0, \vec{P}(x, y, t{-}1)_i + U_{ij}),
                           & |e(x, y, t{-}1)_j| {\ge} M,
                             e(x, y, t{-}1)_j {>} 0,
                             z {\le} \gamma,
                             U_{ij} {<} 0 \\
                         \min(\vec{P}(x, y, t{-}1)_i + D_{ij}, 1), 
                           & |e(x, y, t{-}1)_j| {\ge} M,
                             e(x, y, t{-}1)_j {<} 0,
                             z {\le} \gamma,
                             D_{ij} {>} 0 \\
                         \max(0, \vec{P}(x, y, t{-}1)_i + D_{ij}), 
                           & |e(x, y, t{-}1)_j| {\ge} M,
                             e(x, y, t{-}1)_j {<} 0,
                             z {\le} \gamma,
                             D_{ij} {<} 0 \\
                        \end{cases}
\end{align*},
where $z {\in} Z(0, 1)$ with $Z$ being a random variable. 

\end{document}